### ISSUE TITLE
DCP backfill tracking error message in shadower tests

### ISSUE THREAD

**Issue by adamcfraser** [couchbase/sync_gateway#3321]
The shadower tests sometimes log the following warning when run against a Couchbase server bucket:

```
2018-02-16T16:56:17.232-08:00 WARNING: Invalid range provided for DCP backfill tracking - tracking disabled. len(start):1024 len(end):3 -- base.(*backfillStatus).init() at dcp_feed.go:643
```

Seen in http://uberjenkins.sc.couchbase.com/view/Build/job/sync-gateway-integration-master/406/consoleFull

On DCP startup, the set of high sequence values isn't necessarily the full vbucket set (empty vbuckets are not included).  It looks like DCP backfill doesn't handle this scenario, at least in the Shadowing case.  Need to review whether this is a shadowing only issue, and if so, decide how best to fix.

**Comment by adamcfraser** [comment]
Needs review of severity to determine whether it has to be fixed for 2.0

**Comment by adamcfraser** [comment]
Seeing the same backfill logging error during continuous import, if the bucket doesn't have data in all vbuckets.
